README.md

```diff
@@ -4,64 +4,85 @@
+![Homepage Screenshot](/public/images/homepage.png "Antirecurso Homepage")
 
-Here's a brief overview of each component in the TALL stack for Laravel:
+2. Register an account or login if you already have one.
 
-- Tailwind CSS: Tailwind CSS is a CSS framework that provides a set of utility classes for building user interfaces. It allows you to build layouts and style your application using a set of predefined classes, rather than writing your own custom CSS.
+3. Choose your desired subject from the wide array available.
 
-- Alpine.js: Alpine.js is a small JavaScript library that provides reactive components and two-way data binding for building user interfaces. It is often used as an alternative to larger JavaScript frameworks like React or Vue.js.
+![Homepage Screenshot](/public/images/desiredSubject.png "Antirecurso Homepage")
 
-- Laravel Livewire: Laravel Livewire is a PHP library that makes it easy to build dynamic, interactive UIs with Laravel. It allows you to create components that can automatically update the DOM without the need for full page reloads.
+4. Choose the Type of questions you wish to take
 
-- Laravel: Laravel is a popular PHP framework for building web applications. It provides a set of tools and features that make it easier to build complex, feature-rich applications. Laravel is known for its clean codebase, excellent documentation, and strong developer community.
+![Homepage Screenshot](/public/images/typeOfQuestions.png "Antirecurso Homepage")
 
-Together, these technologies can be used to build powerful, dynamic web applications that you can use in the comfort of your house 🤗.
+5. Submit your answers, and immediately receive feedback on your performance, including a breakdown of correct and incorrect responses.
 
-## Releases 📑
+6. Track your progress over time using the inbuilt scoreboard feature.
 
-For more detailed information about the releases, you can check our <a href="https://github.com/Nucleo-Estudantes-Informatica-ISEP/antirecurso/releases" target="_blank">releases' page</a>, but here's a little preview table!
+![Homepage Screenshot](/public/images/score.png "Antirecurso Homepage")
+
+
+## The Tech Stack 🔧
+
+For this project, we've transitioned to using the Next.js framework. It brings productivity and agility in development and allows us to build server-side rendering and static web applications.
+
+Here's a quick rundown of the tech stack:
+
+- **Next.js**: A top-tier open-source React front-end development web framework that enables functionality such as server-side rendering and generating static websites for React based web applications.
+
+- **Tailwind CSS**: A utility-first CSS framework packed with classes like flex, pt-4, text-center and rotate-90 that can be composed to build any design, directly in your markup.
+
+- **Alpine.js**: A rugged, minimal framework for composing JavaScript behavior in your markup. It’s like Tailwind for JavaScript.
```

Comment:
We do not use Alpine.js anymore; you could instead add a reference to Framer Motion, for example.